### PR TITLE
cli 0 -> 1

### PR DESCRIPTION
**Pull request by scgbckbone**
* cli now now uses range(1, 11) instead of range(10) for slots
* fix tor version for GH actions (again)
* adjust README
* fix simulator (active slot should never be over 9)
* fix wif command to show active slot unsealed or active - 1
* get_address workaround for last unsealed slot

### CODE REVIEW

File: cktap/cli.py [hunk 1]
```diff
@@ -655,12 +672,14 @@ def export_to_core(cvc, pretty, slot):
         label=""  # label will be set to slot number
     )
     descriptor_list = []
-    for _slot in range(card.active_slot+1):
-        if slot and _slot not in slot:
+    for be_slot in range(card.active_slot+1):
+        if be_slot > 9:
```
Comment (scgbckbone) [inline cktap/cli.py:676]: this is actually a bug that is also present in master... but imo correct way to fix this is to not let `card.active_slot` (on simulator) to reach number 10 --> have not tested real card yet so not sure about difference there

active_slot is 10 when all slots are unsealed

File: cktap/proto.py [hunk 2]
```diff
@@ -328,10 +328,10 @@ def unseal_slot(self, cvc):
         resp = self.send('dump', slot=target)
 
         if resp.get('used', None) == False:
-            raise RuntimeError(f"Slot {target} has not been used yet.")
+            raise RuntimeError(f"Slot has not been used yet. Use 'setup' cmd first.")
```
Comment (scgbckbone) [inline cktap/proto.py:332]: do not print slot number as it is different on be and fe now

File: cktap/cli.py (hunk 1 above)
Comment (doc-hex) [reply cktap/cli.py:676]: yeah lets make the emulator do what the card does (not sure what that is)

Comment (scgbckbone) [reply cktap/cli.py]: tested with `1.0.0`

**Pull request merged**